### ISSUE TITLE
[BUG] No Tools Listed Under Tool Proficiencies DND5e 2.1.5

### ISSUE THREAD

**Issue by arsserge** [sdenec/tidy5e-sheet#770]
v0.9.2

**Describe the bug**
No Tools showing in Tool Proficiency. Cause is update to DND 2.2.x which relocated the toolprof under tools.

**Screenshots**
https://user-images.githubusercontent.com/49371326/243140225-f110f46e-f083-430f-8e7b-77b08916f729.png

**Browser:**
Chrome (latest)

**Foundry Version:** 
Latest 

**Game System:** 
2.1.5

**Comment by szefo09** [comment]
That's true, 0.9.X releases support the D&D 2.2.X Game systems. Use 0.8.41 for older system. @arsserge

**Comment by arsserge** [comment]
just for the tools? r u mad? :D

**Comment by p4535992** [comment]
Should be fixed again on 0.9.17
